src/helpers/spatialHelper.ts

```diff
@@ -21,3 +22,56 @@ export const englandAndWales: GeographyInfo = {
     ],
   },
 };
+
+// uk bounding box, taken from UK bbox value used in API
+// https://github.com/ONSdigital/dp-geodata-api/blob/develop/pkg/geodata/coords.go
+export const UKBbox: Bbox = {
+  east: 1.76,
+  north: 58.64,
+  west: -7.57,
+  south: 49.92,
+};
+
+/*
+  Snap map bounding box map tile grid to make bouding box requests cachable.
+*/
+export const snapBboxToMapGrid = (bbox: Bbox, zoom: number) => {
+  console.debug("Snapping bbox:");
+  console.debug(bbox);
+
+  // round down zoom level to ensure snap is to integer tile grid
+  const floorZoom = Math.floor(zoom);
```

Comment:
Didn't understand this - I've never seen a non-integer zoom?

Reply:
might be a mouse-wheel thing